cmd/lncli/commands.go

```diff
@@ -1355,6 +1355,63 @@ func unlock(ctx *cli.Context) error {
 	return nil
 }
 
+var changePasswordCommand = cli.Command{
+	Name: "changepassword",
```

Comment:
Must be rebased and added to correct command group :)

Reply:
Fixed.